docs/pages/get-started/connect.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: "Step 2 - Connect Infrastructure"
+description: "Enroll the resources you want to protect with your Teleport cluster."
+tocDepth: 2
+tags:
+  - get-started
+---
+
+import Button from '@site/src/components/Button';
+import Icon from '@site/src/components/Icon';
+
+After deploying your Teleport cluster, the next step is to enroll the infrastructure resources you want to secure.
+
+Teleport supports a wide range of resource types, including Linux servers, MCP servers, Kubernetes clusters, databases, internal web apps, and cloud provider APIs. Each enrolled resource connects back to your cluster via a secure reverse tunnel, enabling identity-aware access with full session visibility and control.
+
+In this step, we'll walk through enrolling an Ubuntu Linux server as your first resource.
+
+## Prerequisites
+
+You'll need one of the following:
+
+- An Ubuntu Linux server that you can SSH into (physical server, VM or cloud instance like EC2, or homelab server)
```

Comment:
Right now, it seems like the Prerequisites section includes requirements for the whole guide, and that the guide is for connecting infrastructure in general. In other words, it seems like the user needs an Ubuntu Linux server and Docker to connect infrastructure in general. 

I would make it clearer that there's a demo/getting started workflow that the user can follow as a quick example.

Also, for a new user, there's only guidance for enrolling this Ubuntu server. We should add some more explicit mention of the fact that there are other resources a user can add through the Web UI.